src/Microsoft.OData.Client/IDataServiceQueryOfT.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Threading.Tasks;
+
+namespace Microsoft.OData.Client
+{
+    public interface IDataServiceQueryOfT<TElement>

```

Comment:
```suggestion
    public interface IDataServiceQuery<TElement>
```

Reply:
The interfaces are to be used by the developer to test his code as opposed to unit tests in Odata client libraries.